<p><img alt="caee logo" src="images/caee_logo.png" align="left" hspace="10px" vspace="0px"></p>
<p><img alt="iel logo" src="images/iel_logo.png" align="right" hspace="10px" vspace="0px"></p>
<h1 align='center'>Homework 1</h1>

ARE 346P / CE 389H HVAC Design

Professor Dr. Zoltan Nagy

**Due: September 19, 2020**

<h2>Instructions</h2>

Solve each of the following problems and show your work. Make sure to label each answer with the same label as the problem.

You will be graded on both the correct method and solution. Partial credit is awarded for use of the correct equation or method, correct values chosen from tables and charts when required, and the correct answer with appropriate units. 

This assignment includes *five* problems based on Chapter 7 from your textbook.

| PROBLEM | POINTS |
| --- | --- |
| 7.1 | 8 pts |
| 7.21 | 4 pts |
| 7.3 | 4 pts |
| 7.4 | 4 pts |
| 7.27 | 5 pts |
| **TOTAL** | **25 pts** |

<h2>Navigation</h2>

Click on the hyperlink below to go to the problem you are interested in:

- [7.1](#7.1)
- [7.21](#7.21)
- [7.3](#7.3)
- [7.4](#7.4)
- [7.27](#7.27)

<h2>7.1</h2>

Moist air exists at 80$^\circ$F dry-bulb temperature, 0.0150 lbm$_\text{w}$/lbm$_\text{a}$ humidity ratio, and 14.696 psia pressure. Through the use of <u>Table A.4 and fundamental relations</u>, determine:

   (a) the dew-point temperature, <br>
   (b) relative humidity, <br>
   (c) volume in ft$^3$/lbm$_a$, and <br>
   (d) the enthalpy in Btu/lbm$_a$ <br>

**Note**: Do not use the psychrometric chart for this problem

<h3>7.1a</h3>

Using <u>Table A4E</u> with $W = 0.0150 \text{ lbm}_\text{w}$:

<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{t}_\text{d} = 68.4 ^\circ \text{F} $
</div>

<h3>7.1b</h3>

Using <u>Table A4E</u> with $ DBT = 80^\circ \text{F} $:
 - $ \text{W}_\text{s} = 0.022340 \text{ lbm}_\text{w}/\text{lbm}_\text{a}$:

By Eq. 7.6 we get:

$$
\begin{align}
\phi &= \frac{ 1 + \frac{0.622}{W_s} }{ 1 + \frac{0.622}{W} } \\
&= \frac{ 1 + \frac{0.622}{0.02234} }{ 1 + \frac{0.622}{0.0150} }
\end{align}
$$


In [5]:
# defining humidity ratios
W = 0.0150
Ws = 0.022340

# solving with eq. 7.6
phi = (1 + (0.622 / Ws)) / (1 + (0.622 / W))

print(f'Relative Humidity: {phi*100}')

Relative Humidity: 67.91782204239041


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{RH} = 67.9\% $
</div>

<h3>7.1c</h3>

Using <u>Table A4E</u> with $ DBT = 80^\circ \text{F} $:
- $ \text{v}_\text{a} = 13.602 \text{ ft}^3/\text{lbm}_a $
- $ \text{v}_\text{as} = 0.487 \text{ ft}^3/\text{lbm}_a $

By Eq. 7.15 we get: 

$$ 
\begin{align}
v &= v_a + \mu v_{as} \\
&= v_a + \frac{W}{W_s} v_{as} \\
&= 13.602 + \frac{0.0150}{0.02234} \left( 0.487 \right)
\end{align}
$$

In [7]:
# defining specific volumes
va = 13.602
vas = 0.487

# calculating degree of saturation
mu = W / Ws

# solving with eq. 7.15
v = va + mu * vas

print(f'Specific Volume: {v}')

Specific Volume: 13.928991942703671


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{v} = 13.929 \text{ ft}^3/\text{lbm}_a$
</div>

<h3>7.1d</h3>

Using <u>Table A4E</u> with $ DBT = 80^\circ \text{F} $:
- $ \text{h}_\text{a} = 19.222 \text{ Btu}/\text{lbm}_a $
- $ \text{h}_\text{as} = 24.479 \text{ Btu}/\text{lbm}_a $

By Eq. 7.16 we get: 

$$ 
\begin{align}
h &= h_a + \mu h_{as} \\
&= h_a + \frac{W}{W_s} h_{as} \\
&= 19.222 + \frac{0.0150}{0.02234} \left( 24.479 \right)
\end{align}
$$

In [10]:
# defining enthalpies
ha = 19.222
has = 24.479

# solving with eq. 7.16
h = ha + mu * has

print(f'Specific Enthalpy: {h}')

Specific Enthalpy: 35.658213070725154


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{h} = 36.658 \text{ Btu}/\text{lbm}_a$
</div>

<h2>7.21</h2>

Re-work Problem [7.1](#7.1) again but using the psychrometric chart. Compare your answers to those you found in 7.1. Write out the answers and a brief explanation for each part.

   (a) the dew-point temperature, <br>
   (b) relative humidity, <br>
   (c) volume in ft$^3$/lbm$_a$, and <br>
   (d) the enthalpy in Btu/lbm$_a$ <br>


<h3>7.1a</h3>

<h3>7.1b</h3>

<h3>7.1c</h3>

<h3>7.1d</h3>

<h2>7.3</h2>
    
The atmosphere within a room is at 70$^\circ$F dry-bulb temperature, 50% degree of saturation, and 14.696 psia pressure. The inside surface temperature of the windows is 40$^\circ$F. Will moisture condense out of the air upon the window glass? Show your work and explain your reasoning.

<h2>7.4</h2>

Assume that the dimensions of the room of Problem [7.3](#7.3) are 30 ft by 15 ft by 8 ft high. Calculate the number of pounds of water vapor in the room.

<h2>7.27</h2>

In the psychrometric process, moist air at standard atmospheric pressure goes from state 1 at 15$^\circ$C dry-bulb temperature and 10$^\circ$C wet-bulb temperature to state 2 of 35$^\circ$C dry-bulb temperature and 10$^\circ$C dew-point temperature. By using the psychrometric chart, determine:

   (a) the enthalpy-humidity ratio, q’, for the process, and <br>
   (b) the sensible-heat ratio, SHR, for the process

<h3>7.27a</h3>

<h3>7.27b</h3>